di-ipv-orchestrator-stub/deploy/template.yaml

```diff
@@ -481,7 +481,7 @@ Resources:
   OrchStubAutoScalingTarget:
     Type: AWS::ApplicationAutoScaling::ScalableTarget
     Properties:
-      MaxCapacity: 1
+      MaxCapacity: 8
```

Comment:
I think this was overruling the 'desired task count' for the service, so there was still only a single task